background.js

```diff
@@ -72,6 +72,20 @@ const DEFAULT_NO_SKIP_URLS_LIST = [
     "/subscribe",
     "/unauthenticated",
     "/verification",
+    "//bugs.chromium.org/",
+    "//calendar.google.com/",
+    "//downforeveryoneorjustme.com/",
+    "//outlook.office.com/",
+    "//safebrowsing.google.com/",
+    "//twitter.com/intent/",
+    "//uc.appengine.google.com/",
+    "//web.archive.org/",
+    "//www.facebook.com/sharer/",
+    "//www.facebook.com/sharer/",
+    "//www.linkedin.com/cws/",
```

Comment:
Personally I wouldn't like to have by default skips for facebook, google, twitter or outlook, IMO the extension should block all redirects by default.    

If you'd like to provide this list of "include these if you are having issues on these sites" it should be optional, maybe in a FAQ section for people to copy and paste or a separate list to toggle.